packages/wallets/src/features/cfd/flows/Verification/Verification.tsx

```diff
@@ -81,6 +82,7 @@ const Verification: FC<TVerificationProps> = ({ selectedJurisdiction }) => {
     const { data: settings, update: updateSettings } = useSettings();
     const { submitIDVDocuments } = useIdentityDocumentVerificationAdd();
     const { getModalState, hide, show } = useModal();
+    const { isMobile } = useDevice();
```

Comment:
can't wait for when we gonna be able to use css / media queries or something else than javascript to determine layout concerns :D